code/game/antagonist/outsider/raider_mage.dm

```diff
@@ -0,0 +1,63 @@
+var/datum/antagonist/raider_mages/raider_mages
+
+/datum/antagonist/raider_mages
```

Comment:
```suggestion
/datum/antagonist/raider_mage
```
The name of used in the antag datums is always singular
(Make sure to update the other occurrence as well)